### PR TITLE
Review Past project copied repo

### PR DESCRIPTION
**Pull request by tahseenzahra**
This branch includes all the files of the past project that was done with the collaboration of fellow during microverse admission trial.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <p>this section contains the contact us information </p>
+      <a href="https://github.com/tahseenzahra"><p>github</p></a>
+      
+
+    </div>
+    <footer id="footer">
+      <div>cretaed for microverse collaborative project</div>
+    </footer>
+
+  </body>
```
Comment (tahseenzahra) [inline index.html:34]: Please remove the unnecessary new lines. As they make the code unclean.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    display: flex;
+    flex-direction: column;
+}
+.navbar{
+    display: flex;
+    align-items: center;
+    justify-content: right;
+    position: fixed;
+
+}
```
Comment (tahseenzahra) [inline styles.css:22]: Kindly just add two space indentations before declaring the property, add a new line before giving the style to anything (like on line 16 new line should be added), and remove empty lines.

File: README.md [hunk 3]
```diff
@@ -1 +1 @@
-# Review-one-of-past-projects
\ No newline at end of file
+# microverse_2nd_collab
```
Comment (tahseenzahra) [inline README.md:1]: Kindly add some information about the languages you used and about collaborators. Readme file should be descriptive about the project.

**Review by tahseenzahra — COMMENTED** [review]
# Required Changes :recycle: 

Please check my comments below and make the required changes.

Submit for re-review when you are done.

Good Luck :blush:

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @tahseenzahra 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- You wrote clear comments without typos ✔️ 
- You wrote polite suggestions 😄 
- You gave reasons why a certain change is required 🏄 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- It would be nice if you add a couple of emojis in some of your comments 😎 . There is a saying that goes, "An emoji is worth 20 words". Emojis make the requests more enjoyable to read and I think it would be a good idea 🏄

**Pull request merged**